packages/boot/src/booters/base-artifact.booter.ts

```diff
@@ -37,6 +38,9 @@ export class BaseArtifactBooter implements Booter {
    * Project root relative to which all other paths are resolved
    */
   readonly projectRoot: string;
+
+  private packageRoot: string;
```

Comment:
Please add a tsdoc comment to describe this new property and update both `projectRoot` and `packageRoot` comments to explain how are these two properties different from each other.